### DISCUSSION TITLE
Possible contribution

### DISCUSSION THREAD

**Discussion by Amauvp** [sherlock-project/sherlock#1598] [General]
Hello everyone, 

I am currently a first year student of a Master in Cybersecurity at the University of Brussels. As part of an Open Source Strategy course, I'm asked to make a small contribution to an open source project. This project has attracted my curiosity, would it be possible to make a small contribution to it? This contribution does not have to be a new feature but can be a slight update of the documentation, a modification of the display or other. 

Thank you in advance for your answer.

**Comment by Ngostary2k** [comment]
Hai,  I like your conversation here. I am also new here and would like ,akr contributions to a project too. I am searching for a project to contribute it. Someone can help me please?

**Comment by rskbansal** [comment]
> Hello everyone,
> 
> I am currently a first year student of a Master in Cybersecurity at the University of Brussels. As part of an Open Source Strategy course, I'm asked to make a small contribution to an open source project. This project has attracted my curiosity, would it be possible to make a small contribution to it? This contribution does not have to be a new feature but can be a slight update of the documentation, a modification of the display or other.
> 
> Thank you in advance for your answer.

@Amauvp @Ngostary2k Yes, its a good project to get started to be honest. I myself added 2 sites which were previously removed and now works like a charm. I am not an experienced programmer or something but I guess reading the [CONTRIBUTING.md](https://github.com/sherlock-project/sherlock/blob/master/CONTRIBUTING.md) document, you can easily figure out what's need to be done. You can even correct some typos, if you find any or just document things properly, even that's important. So, there a lot of opportunities to learn, just make your first Pull Request. Cheers!
